### PR TITLE
Sh rejig about team page

### PR DESCRIPTION
**Pull request by SamHollings**
# Description

❓**What**: Reformatted about team page a bit

🧠**Why?**: to make it look a bit more attractive and move "contact" and "our work" to be more prominent

👨‍💻**How?**: changing the about our team page

# Checklist:
Have checked for the following:
- [x] The website still builds correctly, and you can view it using `mkdocs serve`.
- [x] There are no new "warnings" from mkdocs
- [x] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [x] Spelling errors
- [x] Consistent capitalization
- [x] Consistent numbers
- [x] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [x] Code snippets don't work
- [x] Images not working
- [x] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive
- devcontainer.json describes further settings

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -5,21 +5,54 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
+
+
+We are the **[NHS England](https://www.england.nhs.uk/) Data Science Team**.
```
Comment (amaiaita) [inline docs/about.md:13]: image
im not in love with how the difference in boldness looks

File: docs/about.md [hunk 2]
```diff
@@ -5,21 +5,54 @@ hide:
...
+[See our work](./our_work/index.md){ .md-button}
+
+
+</div>
 
 ## How are we different from analytical teams?
-**Do** – We develop and deploy data science products to make a positive impact on NHS patients and workforce. ​​
 
-**Delve** - We investigate applying novel techniques that increase the insight we get from health-related data.​
+<div class="grid cards" markdown>
```
Comment (amaiaita) [inline docs/about.md:29]: this is great! love it

File: docs/about.md [hunk 3]
```diff
@@ -5,21 +5,54 @@ hide:
...
+
+We are the **[NHS England](https://www.england.nhs.uk/) Data Science Team**. 
+
+**Our vision is**: 
 
 > For NHS England to lead on embracing data science in the NHS for the betterment of all our patients and all our staff​. 
 
-<img src=../images/DS_team_photo.jpeg alt-tex="Picture of the Data Science team stood on some steps in London." width="600" >
-</img>
+
```
Comment (amaiaita) [inline docs/about.md:19]: an extra space between our vision and the contact us button would look better i think, currently theyre a little too squished together.

File: docs/about.md [hunk 4]
```diff
@@ -5,21 +5,54 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
```
Comment (amaiaita) [inline docs/about.md:10]: do you reckon youd be able to compress the image so it loads better? Otherwise I'll do it at some point

Comment (SamHollings) [reply docs/about.md:10]: Done

File: docs/about.md (hunk 3 above)
Comment (SamHollings) [reply docs/about.md:19]: Added!

File: docs/about.md [hunk 5]
```diff
@@ -5,21 +5,56 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo_smaller.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
+
+
+We are the **[NHS England](https://www.england.nhs.uk/) Data Science Team**.
```
Comment (SamHollings) [inline docs/about.md:13]:
```suggestion
We are the [NHS England](https://www.england.nhs.uk/) Data Science Team. 
```

File: docs/about.md [hunk 6]
```diff
@@ -5,21 +5,56 @@ hide:
...
+
+**Our vision is**: 
 
 > For NHS England to lead on embracing data science in the NHS for the betterment of all our patients and all our staff​. 
 
-<img src=../images/DS_team_photo.jpeg alt-tex="Picture of the Data Science team stood on some steps in London." width="600" >
-</img>
+<br/>
+
+[Contact Us](mailto:datascience@nhs.net){ .md-button .md-button--primary }
```
Comment (amaiaita) [inline docs/about.md:21]: this button doesnt do anything when you click it :'(

**Review by amaiaita — CHANGES_REQUESTED** [review]
just the contact us button not working - everything else looks great!

File: docs/about.md (hunk 6 above)
Comment (SamHollings) [reply docs/about.md:21]: Resolved to some extent!

**Review by amaiaita — APPROVED** [review]

**Pull request merged**